crypto/test/sysgenid_test_util.cc

```diff
@@ -0,0 +1,97 @@
+// -----------------------------------------------------------------------------
```

Comment:
Why did you choose this location for this file and not for example under rand_extra?